src/frontend/web_application/src/scenes/MessageList/index.js

```diff
@@ -0,0 +1,30 @@
+import { createSelector } from 'reselect';
+import { bindActionCreators, compose } from 'redux';
+import { connect } from 'react-redux';
+import { withTranslator } from '@gandi/react-translate';
+import { requestMessages, updateMessage } from '../../store/modules/message';
+import Presenter from './presenter';
+
+const messagesSelector = state => state.message.messages;
+const messageByIdSelector = state => state.message.messagesById;
```

Comment:
no, the state has a list of messageIds and an object w/ messages referenced by their id. In this case it is not much useful but it is consistent with discussions, tags etc.